### ISSUE TITLE
Errors when using a client not based on docker-py

### ISSUE THREAD

**Issue by merll** [merll/docker-map#12]
[Problem occurs in Docker-Fabric](https://github.com/merll/docker-fabric/issues/10), but is caused by a check if the client is a subclass of ``docker.client.Client``. This check should probably be replaced with something more flexible in order to support non-API clients such as the CLI.

**Comment by ambsw-technology** [comment]
The following worked for me (and still respects a Config vs. client distinction:

```
            if isinstance(docker_client, ClientConfiguration):
                default_client = docker_client
            else:
                default_client = self.configuration_class.from_client(docker_client)
```

**Comment by merll** [comment]
I agree that this one check for ``ClientConfiguration`` is sufficient. This will be updated in the next release.

**Comment by merll** [comment]
Fixed in Docker-Map release `0.7.1`.
